addons/im_livechat/static/tests/messaging_service_patch.test.js

```diff
@@ -1,62 +0,0 @@
-import { contains, onRpcBefore, start, startServer } from "@mail/../tests/mail_test_helpers";
-import { withGuest } from "@mail/../tests/mock_server/mail_mock_server";
-import { describe, test } from "@odoo/hoot";
-import {
-    asyncStep,
-    Command,
-    mockService,
-    serverState,
-    waitForSteps,
-} from "@web/../tests/web_test_helpers";
-
-import { rpc } from "@web/core/network/rpc";
-import { defineLivechatModels } from "./livechat_test_helpers";
-
-describe.current.tags("desktop");
-defineLivechatModels();
-
-test("Notify message received out of focus", async () => {
-    const pyEnv = await startServer();
-    const guestId = pyEnv["mail.guest"].create({ name: "Visitor" });
-    const channelId = pyEnv["discuss.channel"].create({
-        name: "Livechat 1",
-        channel_type: "livechat",
-        channel_member_ids: [
-            Command.create({ partner_id: serverState.partnerId }),
-            Command.create({ guest_id: guestId }),
-        ],
-    });
-    onRpcBefore("/mail/data", async (args) => {
-        if (args.init_messaging) {
-            asyncStep(`/mail/data - ${JSON.stringify(args)}`);
-        }
-    });
-    mockService("presence", { isOdooFocused: () => false });
-    await start();
-    await waitForSteps([
-        `/mail/data - ${JSON.stringify({
-            init_messaging: {},
-            failures: true,
-            systray_get_activities: true,
-            context: {
-                lang: "en",
-                tz: "taht",
-                uid: serverState.userId,
-                allowed_company_ids: [1],
-            },
-        })}`,
-    ]);
-    // send after init_messaging because bus subscription is done after init_messaging
-    await withGuest(guestId, () =>
-        rpc("/mail/message/post", {
-            post_data: {
-                body: "Hello",
-                message_type: "comment",
-                subtype_xmlid: "mail.mt_comment",
-            },
-            thread_model: "discuss.channel",
-            thread_id: channelId,
-        })
-    );
-    await contains(".o_notification:has(.o_notification_bar.bg-info)", { text: "Hello" });
-});
```

Comment:
Should keep the test but adapt for native notification.

Also very few code has been removed, because we need toaster for Android. Would be nice to put test coverage of Android to justify non-removal of this code.